docs/UserGuide.md

```diff
@@ -193,6 +194,23 @@ Examples:
 
 [return to top <img src="images/toc-icon.png" width="25px">](#table-of-contents)
 
+---
+### Editing a comment: `comment`
+
+Edit comment of a specific show from Trackermon.
+
+Format: `comment <INDEX> [c/<COMMENT>]`
+* Edit comment of the show at the specified `<INDEX>`.
+* The index refers to the index number shown in the displayed show list. (not overall)
+* The index **must be a positive integer** 1,2,3,...
+* Omitting the `[c/<COMMENT>]` would remove the comment of that specific show.
+
+Examples:
+* `list` followed by `comment 2 c/Shags` edit 2nd show's comment in Trackermon to "Shags".
```

Comment:
Should this be `edits the` instead of `edit`?

Reply:
I will edit it ASAP.